Tella/Data/Database/ResourceDatabase.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  ResourceDatabase.swift
+//  Tella
+//
+//  Created by gus valbuena on 2/15/24.
+//  Copyright © 2024 HORIZONTAL. All rights reserved.
+//
+
+import Foundation
+
+extension TellaDataBase {
+    func createResourceTable() {
+        let columns = [
+            cddl(D.cId, D.text, primaryKey: true, autoIncrement: false),
+            cddl(D.cExternalId, D.text),
+            cddl(D.cFilename, D.text),
+            cddl(D.cTitle, D.cTitle),
+            cddl(D.cSize, D.text),
+            cddl(D.cCreatedDate, D.text),
+            cddl(D.cServerId, D.integer, tableName: D.tServer, referenceKey: D.cServerId)
+        ]
+        
+        statementBuilder.createTable(tableName: D.tResource, columns: columns)
+    }
+    
+    func getDownloadedResources() -> [DownloadedResource] {
+        
+        var resources: [DownloadedResource] = []
+        
+        do {
+            let responseDict = try statementBuilder.selectQuery(tableName: D.tResource)
+            
+            responseDict.forEach { dict in
+                resources.append(getResource(dictionnary: dict))
```

Comment:
I think it's better to use `decode` function in Array extension 
see `feedbackDict.decode(Feedback.self) `